src/seedu/addressbook/data/person/Address.java

```diff
@@ -35,6 +37,11 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
     public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
+    
+    @Override
+    public String getPrintableString() {
+    	return LABEL + this.toString();
+    }
```

Comment:
Shouldn't you check if it's a private or not? The variable name `LABEL` is too generic.